.github/workflows/clippy-fmt.yml

```diff
@@ -13,7 +13,17 @@ jobs:
           toolchain: nightly
           components: clippy
           override: true
-      - uses: actions-rs/clippy-check@v1
+      - name: Check with Clippy
+        uses: actions-rs/clippy-check@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           args: --all-features --all --tests
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          components: rustfmt
+          override: true
+      - name: Check with rustfmt
+        uses: actions-rs/cargo@v1
+        with:
+          command: fmt --all -- --check
```

Comment:
Since this is a much faster check than clippy, I'd say put it first it fails quicker and doesn't install crates.

Reply:
Indeed! Will do so.